tests/deb/pkg_deb_test.py

```diff
@@ -158,6 +158,7 @@ def test_description(self):
 	'Replaces: oldpkg',
 	'Breaks: oldbrokenpkg',
 	'Provides: hello',
+  'License: Apache-2.0',
```

Comment:
The indent looks off.

Reply:
:+1: there were a couple of tabs in the file - took the liberty and changed them to spaces (looks like spaces are commonly used for indentation).